packages/manager/modules/pci/src/projects/project/storages/databases/databases.controller.js

```diff
@@ -38,4 +38,62 @@ export default class {
     this.trackDatabases('table::options_menu::delete_database');
     this.goToDeleteDatabase(database);
   }
+
+  renameDatabase(database) {
+    this.trackDatabases('table::options_menu::rename_database');
```

Comment:
extract prefix to a constant 'table::options_menu::' to avoid duplication

Reply:
Done with https://github.com/ovh/manager/commit/fcd5037286f4bf8daffb8e00ebf54c5ee561d9b3